src/components/icons/icons.xml

```diff
@@ -955,4 +924,28 @@ https://fontawesome.com/license -->
       <circle cx="2" cy="12.5" r="1"/>
     </svg>
   </t>
+  <t t-name="o-spreadsheet-Icon.EDIT_TABLE">
+    <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18">
```

Comment:
the margin on the side is less than a pixe, which I thinkg is rounded to 0, it looks like the styles is broken. Could it be possible to crop it a bit horizontally?

Reply:
Added a bit of  apadding in the ActionButton. It sounds a bit dumb to have a space of 18px for icons, but that we cannot actually use icons of 18px because we need a padding inside the icon...